CMakeLists.txt

```diff
@@ -333,3 +345,10 @@ install(DIRECTORY ${ONNX_ROOT}/onnx DESTINATION include
 install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/onnx DESTINATION include
     FILES_MATCHING
     PATTERN "*.h")
+if(ONNX_BUILD_TESTS)
+    include_directories(
```

Comment:
Do we still need to add these include dirs at the top level? You are already doing `target_include_directories` in unittest.cmake.

Reply:
yes.. that's the minimal set working on windows

Reply:
sorry i misunderstand ur point.. moved the include dirs from top level to target_include_directories